docs/jwst/references_general/abvega_offset_selection.inc

```diff
@@ -0,0 +1,16 @@
+Reference Selection Keywords for ABVEGA_OFFSET
```

Comment:
Need to add ".. _abvega_offset_selectors:" reference at the top of this file.

Reply:
This page isn't referenced (i.e., using `:ref:` anywhere).  It's included (`.. include::`) in the `abvega_offset_reffile` docs.  Does it still need a reference tag?

Reply:
Perhaps not. We have it all the other ref file docs, but you're right that if the reference is never used anywhere, it's obviously not necessary.